SubMotionVR Mod/SubMotion/SubMotion/Patchers/ArmsController.cs

```diff
@@ -98,29 +95,29 @@ public void Postfix(ArmsController __instance, FullBodyBipedChain ik, PlayerTool
     [HarmonyPatch("Reconfigure")]
     internal class ArmsController_Reconfigure_Patch
     {
-
         [HarmonyPrefix]
-        public void Prefix(ArmsController __instance, PlayerTool tool)
+        public void Prefix(ArmsController __instance, PlayerTool tool, FullBodyBipedIK ___ik, PDA ___pda, Transform ___leftWorldTarget, Transform ___rightWorldTarget)
         {
-            __instance.ik.solver.GetBendConstraint(FullBodyBipedChain.LeftArm).bendGoal = __instance.leftHandElbow;
-            __instance.ik.solver.GetBendConstraint(FullBodyBipedChain.LeftArm).weight = 1f;
+            ___ik.solver.GetBendConstraint(FullBodyBipedChain.LeftArm).bendGoal = __instance.leftHandElbow;
+            ___ik.solver.GetBendConstraint(FullBodyBipedChain.LeftArm).weight = 1f;
             if (tool == null)
             {
-                __instance.leftAim.shouldAim = false;
-                __instance.rightAim.shouldAim = false;
-                __instance.ik.solver.leftHandEffector.target = null;
-                __instance.ik.solver.rightHandEffector.target = null;
-                if (!__instance.pda.isActiveAndEnabled)
+                Traverse.Create(__instance).Field("leftAim").Field("shouldAim").SetValue(false);

```

Comment:
I'm not 100% certain that this will work as intended, but this is a way of accessing private variables